spec/features/view_archived_petition_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'View archived petition', type: :feature do
+  let(:archive_message) {'This petition has been archived It was submitted during the 2010–2015 Conservative – Liberal Democrat coalition government'}
+  let(:petition) {@petition = FactoryGirl.create(:archived_petition, :closed, title: 'Spend more money on Defence')}
+  let(:rejected_petition) {@petition = FactoryGirl.create(:archived_petition, :rejected, title: "Please bring back Eldorado", reason_for_rejection: "<i>We<i> like http://www.google.com and bambi@gmail.com")}
+  
+  it 'that is closed' do
+    visit archived_petition_url(petition)
+    expect(page).to have_content(petition.title)
+    expect(page).to have_content(petition.description)
+    expect(page).to have_title('Spend more money on Defence'.to_s)
+    expect(page).to have_css("li.meta-deadline", :text => "Deadline " + petition.closed_at.strftime("%e %B %Y").squish)
+    expect(page).to have_content(archive_message)
+  end
+  
+  it 'via the old URL' do
+    visit petition_url(petition)
+    expect(current_path).to eq(archived_petition_path(petition))
+    expect(page).to have_content(petition.title)
+    expect(page).to have_content(petition.description)
+    expect(page).to have_title('Spend more money on Defence'.to_s)
+    expect(page).to have_css("li.meta-deadline", :text => "Deadline " + petition.closed_at.strftime("%e %B %Y").squish)
+    expect(page).to have_content(archive_message)
+  end
+  
+  it 'that contains urls, email addresses and html tags' do
+    @petition = FactoryGirl.create(:archived_petition, :closed, title: 'Spend more money on Defence', description: "<i>We<i> like http://www.google.com and bambi@gmail.com")
```

Comment:
It's required to add the description which is used later on in the test, links etc.